### DISCUSSION TITLE
Incomplete releases list from github api

### DISCUSSION THREAD

**Discussion by dr-nyt** [community/community#24720] [Apps, API and Webhooks]
<p>Hey, I’m not sure if this is intended, but the GitHub API won’t list all the releases made on a repository.</p>
<p>For this <a href="https://github.com/JohnCoates/Aerial/releases" rel="noopener nofollow ugc">repository</a> the releases go back to alpha 1.0<br>
but on the <a href="https://api.github.com/repos/JohnCoates/Aerial/releases" rel="noopener nofollow ugc">api</a> they only go as far back as 1.9.1</p>
<p>Is the list truncated on purpose?</p>

**Comment by bartfeenstra** [comment, marked as answer]
<p>For future reference: the releases endpoint is paginated and returns at most 30 releases by default, optionally increased to a 100. See <a href="https://docs.github.com/en/rest/reference/repos#releases" rel="noopener nofollow ugc">https://docs.github.com/en/rest/reference/repos#releases</a> for details, and how to retrieve all releases.</p>
